polkadot-protocol/protocol-components/transactions/transactions-and-block-basis.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Transactions and Block Basis
+description: Overview of transactions and blocks in Polkadot SDK-Based Chains, including signed, unsigned, and inherent transactions.
+---
+
+# Transactions and Block Basis
+
+## Introduction
```

Comment:
Either we should add a note about the relationship between the term Extrinsic and Transaction, or just use Extrinsic IMO